junseungbin/BOJ2439/BOJ2439.java

```diff
@@ -0,0 +1,22 @@
+package assignment;
+
+import java.util.Scanner;
+
+class BOJ2439 {
+    public static void main(String[] args) {
+        int count = 0;
+        Scanner sc = new Scanner(System.in);
+        int limit = sc.nextInt();
+        while(count < limit){
+            count++;
+            for(int n = limit ; n > count ; n--){
+                System.out.print(" ");
+            }
+            for(int m = 0 ; m < count ; m++){
+                System.out.print("*");
+            }
+            System.out.print("\n");
```

Comment:
줄바꿈을 할 때에 `System.out.print("\n");` 대신 `System.out.println();`을 사용하면 더 자연스럽운 코드가 될 수 있어요-! 🙌

```suggestion
            System.out.println();
```